jbpm-designer-client/src/main/java/org/jbpm/designer/client/DesignerViewImpl.java

```diff
@@ -34,6 +34,7 @@ public DesignerViewImpl() {
     public DesignerViewImpl(DesignerWidgetView view) {
         this.designerWidget = new DesignerWidgetPresenter(view);
         initWidget(designerWidget.getView().asWidget());
+        this.publisCallOnReisize(this);
```

Comment:
Hi @tsurdilo , I am not sure, but it seems to me as typo. Did you intended name `publicCallOnReisize` ?